packages/devel/pkg-config/package.mk

```diff
@@ -38,5 +38,5 @@ PKG_CONFIGURE_OPTS_HOST="--disable-silent-rules \
 
 post_makeinstall_host() {
   mkdir -p $SYSROOT_PREFIX/usr/share/aclocal
-    cp pkg.m4 $SYSROOT_PREFIX/usr/share/aclocal
+  cp pkg.m4 $SYSROOT_PREFIX/usr/share/aclocal
```

Comment:
It's a coding style convention. I simply fail to understand the other way around: either you consider there is a hierarchy between mkdir and cp for whatever reason, either you don't and you want to do the cp only if the mkdir don't fail and switch to a conditional if...fi statement around the cp. 

Anyway, it's not a very important PR, so do it as you like. 

Regards.